tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3082,6 +3087,15 @@ def runTest(self):
                 assert (fill_leakout_plus_one(self, src_port_id, dst_port_id,
                                               pkt, int(self.test_params['pg']), asic_type))
 
+            # send packets short of triggering egress drop
```

Comment:
why do we need this ? Also, Can we have this change applicable to chassis only ?

Reply:
Incorporated the comments. Kindly review